src/cmd/netfiles/main.c

```diff
@@ -324,7 +324,7 @@ filethread(void *v)
 	threadsetname("file %s", a->file);
 	w = newwin();
 	winname(w, a->file);
-	winprint(w, "tag", "Get Put Look ");
+	winprint(w, "tag", "Get Put ");
```

Comment:
Is this change deliberate?

Reply:
It's an accident.  Can you remove it from this PR?  I can squash it when I get around to it.

Reply:
Not easily, no, because I'm out of town.